### PR TITLE
Update release note with update tool changes

### PR DESCRIPTION
**Pull request by udda1996**
## Purpose
$title

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -76,6 +76,10 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### Improvements
 
+#### Ballerina Update Tool
+
+- Improved ballerina distribution update commands (`bal dist update`, `bal dist pull latest`).
```
Comment (keizer619) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:81]: @udda1996 Shall we elaborate more by mentioning `dist update` will do a patch version bump and we need to use `latest` for minor version bump

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -76,6 +76,11 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### Improvements
 
+#### Ballerina Update Tool
+
+- Improved `bal dist update` command to update the active distribution to the latest patch version (of the active Swan lake version).
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:81]:
```suggestion
- Improved `bal dist update` command to update the active distribution to the latest patch version (of the active Swan Lake version). 
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -76,6 +76,11 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### Improvements
 
+#### Ballerina Update Tool
+
+- Improved `bal dist update` command to update the active distribution to the latest patch version (of the active Swan lake version). 
+- Improved `bal dist pull latest` command to update the active distribution to the latest Swan Lake version.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:82]:
```suggestion
- Improved the `bal dist pull latest` command to update the active distribution to the latest Swan Lake version.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -76,6 +76,11 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### Improvements
 
+#### Ballerina Update Tool
+
+- Improved the `bal dist update` command to update the active distribution to the latest patch version (of the active Swan Lake version).
```
Comment (keizer619) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:81]:
```suggestion
- Improved the `bal dist update` command to update the active distribution to the latest patch version (of the active Swan Lake Update version). 
```

**Review by keizer619 — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**